dwave/optimization/mathematical.py

```diff
@@ -430,6 +431,29 @@ def multiply(x1: ArraySymbol, x2: ArraySymbol, *xi: ArraySymbol,
     raise RuntimeError("implementated by the op() decorator")
 
 
+def sqrt(x: ArraySymbol) -> SquareRoot:
```

Comment:
tests are failing because you need to add `sqrt` to the `__all__` at the top.

Reply:
Getting this on the mac build:
```
By using the agreetolicense option, you are agreeing that you have run this tool with the license only option and have read and agreed to the terms.
If you do not agree, press CTRL-C and cancel this process immediately.
Rosetta 2 update is not available

Exited with code exit status 1
```

Reply:
transient CI error, let me fix.